_config.yml

```diff
@@ -15,7 +15,7 @@ exclude:
     - 'vendor'
 sass:
     style: compressed
-languages: ["cn", "de", "en", "ru", ]
-default_lang: "en"
+languages: ["de-DE", "en-US", "ru-RU", "zh-CN"]
```

Comment:
This may cause discrimination (region name).

Reply:
e.q:
Traditional Chinese Region, include Hong kong, Taiwan and Macao.
Simplified Chinese Region, include Mainland China, Singapore and Malaysia. 

English Country, include U.S.A., U.K., New Zealand and etc.

Reply:
Want to use BCP-47 standard.

Reply:
https://tools.ietf.org/html/bcp47#appendix-A

Reply:
Yes, that was my concern as well. This might cause discrimination. https://github.com/tc39/tc39.github.io/issues/218#issuecomment-737193646

We can use BCP-47 "simple" variant for languages without significant variation, that is, "de", "en", and "ru" stay as they are. For Chinese we can use "zh-Hans" -- but this may make chinese look like it is separate from the other languages... Maybe I am over thinking this and we can just have `["de", "en", "ru", "zh-Hans"]`?

Reply:
yep

Reply:
we can use additional suffixes only where necessary